api/opds.py

```diff
@@ -118,6 +118,12 @@ def groups_url(self, lane):
     def default_lane_url(self):
         return self.groups_url(None)
 
+    def custom_lane_url(self, lane):
```

Comment:
is there ever going to be a case where you'll want both custom_lane_url and feed_url()? If not, you could make this the QueryGeneratedLane implementation of feed_url().